scripts/apps/authoring/authoring/directives/ItemCarouselDirective.ts

```diff
@@ -331,6 +331,10 @@ export function ItemCarouselDirective(notify) {
                         updated = true;
                     },
                 });
+
+                if (scope.currentIndex != null) {
+                    carousel?.trigger('to.owl.carousel', [scope.currentIndex]);
```

Comment:
seems like `carousel` will be always set here, so no need for `?`

Reply:
I originally added it because it wasn't set when running unit tests and it was a lot of trouble to fix it properly.

Reply:
yep I guess in that watch it was useful, but here it shouldn't be needed